platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -188,10 +204,11 @@ private void wire() {
         pcesWriterWiring
                 .latestDurableSequenceNumberOutput()
                 .solderTo(eventDurabilityNexusWiring.latestDurableSequenceNumber());
-
         signedStateFileManagerWiring
                 .oldestMinimumGenerationOnDiskOutputWire()
                 .solderTo(pcesWriterWiring.minimumAncientIdentifierToStoreInputWire());
+
+        linkedEventIntakeWiring.nonExpiredEventWindowOutput().solderTo(shadowgraphWiring.nonExpiredEventWindowInput());
```

Comment:
I'm curious why this wasn't wired to receive the NonAncientEventWindow like everything else?

Reply:
Until very recently, non-ancient event window didn't have the expired event threshold. Now that the event window does have the expiration threshold, this is wired up like everything else.